mlflow/store/dbfs_artifact_repo.py

```diff
@@ -128,3 +132,11 @@ def list_artifacts(self, path=None):
     def _download_file(self, remote_file_path, local_path):
         self._dbfs_download(output_path=local_path,
                             endpoint=self._get_dbfs_endpoint(remote_file_path))
+
+
+def _get_host_creds_from_default_store():
+    store = utils._get_store()
+    if not isinstance(store, RestStore):
+        raise MlflowException('Failed to get credentials for DBFS; they are read from the ' +
+                              'Databricks CLI credentials or _TRACKING* environment variables.')
```

Comment:
Nit: `MLFLOW_TRACKING*` environment variables? Also looks like we don't document the `MLFLOW_TRACKING*` environment variables anywhere, will create a follow-up to do that

Reply:
Filed https://github.com/mlflow/mlflow/issues/1161